pkg/operator/options/options.go

```diff
@@ -58,16 +59,18 @@ func (s *nodeIdentitiesValue) String() string { return strings.Join(*s, ",") }
 type optionsKey struct{}
 
 type Options struct {
-	ClusterName                    string
-	ClusterEndpoint                string // => APIServerName in bootstrap, except needs to be w/o https/port
-	VMMemoryOverheadPercent        float64
-	ClusterID                      string
+	ClusterName             string
+	ClusterEndpoint         string // => APIServerName in bootstrap, except needs to be w/o https/port
+	VMMemoryOverheadPercent float64
+	ClusterID               string
+
 	KubeletClientTLSBootstrapToken string // => TLSBootstrapToken in bootstrap (may need to be per node/nodepool)
 	SSHPublicKey                   string // ssh.publicKeys.keyData => VM SSH public key // TODO: move to v1alpha2.AKSNodeClass?
-	NetworkPlugin                  string // => NetworkPlugin in bootstrap
```

Comment:
As we add support for selected CNI variants, we also want to make/keep BYO CNI possible - even if we don't fully support it right away. With this in mind (and for better alignment with AKS CNI settings in general), let's keep network plugin setting?

Reply:
Added it all back.